CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+v2.0.0 - 2022-06-08
+----------------------
+
+Feature:
+
+* [#35](https://github.com/godaddy/sample-size/pull/35) - Add implementation for sample size calculator which supports multiple metrics and/or variants
+  * [#30](https://github.com/godaddy/sample-size/pull/30) User experience change: prompt a question on number of variants and enable multiple metrics registration
```

Comment:
```suggestion
 * [#30](https://github.com/godaddy/sample-size/pull/30) User experience change: prompt a question on number of variants and enable multiple metrics registration
```

Reply:
I meant to do this since https://github.com/godaddy/sample-size/pull/30 is part of https://github.com/godaddy/sample-size/pull/35. Do you think it's necessary to emphasize this change? I'm okay with either keeping it this way or only put https://github.com/godaddy/sample-size/pull/35 here